modules/apps/dynamic-data-mapping/dynamic-data-mapping-web/src/main/resources/META-INF/resources/js/ddm_form.js

```diff
@@ -4085,7 +4085,7 @@ AUI.add(
 							var defaultLocale = field.getDefaultLocale();
 
 							availableLanguageIds.forEach((locale) => {
-								if (!localizationMap[locale]) {
+								if (localizationMap[locale] === null) {
```

Comment:
@NemethNorbert could you please explain why it has to be `null` but not `undefined` in this condition?

Reply:
Hi @NemethNorbert, 
On 7.3.x is actually `localizationMap[locale] == null`, which means that the condition will be true if `localizationMap[locale]` is `null` or `undefined`. 
But in this case, the condition will only be true if `localizationMap[locale]` is `null`.

Reply:
Hey @carolmariaabb,

you are right, I changed the `null` to `undefined` as the SF does not allow the usage of `==`
I pushed the change.

Thanks,